app/Models/Achievement.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Proto\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Achievement extends Model
+{
+    protected $table = 'achievements';
+
+    protected $fillable = ['name', 'desc', 'img_file_id', 'tier'];
+
+    public function users()
+    {
+        return $this->belongsToMany('Proto\Models\User', 'achievements_users');
+    }
+
+    public function image()
+    {
+        return $this->belongsTo('Proto\Models\StorageEntry', 'img_file_id');
+    }
+
+    public function achievementOwnership()
+    {
+        return $this->hasMany('Proto\Models\AchievementOwnership');
+    }
+
+    public function current($ismember = true)
```

Comment:
Als dit een method is die alle users returned die deze achievement hebben én member zijn, dan zou ik deze method renamen.